hedera-node/cli-clients/src/main/java/com/hedera/services/cli/sign/AccountBalanceType.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (C) 2020-2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.services.cli.sign;
+
+import com.swirlds.common.stream.StreamType;
+
+/**
+ * Contains properties related to Account Balance file type;
+ * Its constructor is private. Users need to use the singleton to denote this type
+ */
+public final class AccountBalanceType implements StreamType {
```

Comment:
Are account balance files considered to be stream files?

In any case, it seems this class is only being used to determine if a given file has the correct file extension. Unless there are plans to further utilize this class in the future, I would suggest simply doing the check manually in `BalanceFileSignCommand::isFileSupported`

Reply:
I might use some other functions for testing later. Kept it for now. Just in a rush to get this in for tomorrow's build.

Reply:
So are account balance files considered stream files? The fact that multiple required overrides are not applicable makes me think they should not be.

If that is the case, it seems counter productive to make an `AccountBalanceType` which inherits from `StreamType`

Reply:
But if it isn't a stream file in the first place, then `StreamType.isStreamFile()` doesn't make any sense

Reply:
Agreed with Austin

Reply:
@alittley 
StreamType.isStreamFile() is used for file comparison in all other places what kind of file we are receiving. we are comparing the

Reply:
It's being used in cases where the type of file being searched for *is a stream file*.

Why not just check the extension directly, rather than inheriting that functionality from a parent class which doesn't logically make sense?

Reply:
removed StreamType from AccountBalanceType